gift_card.py

```diff
@@ -348,3 +352,146 @@ def parse(cls, report, records, data, localcontext):
         return super(GiftCardReport, cls).parse(
             report, records, data, localcontext
         )
+
+
+class GiftCardRedeemStart(ModelView):
+    "Gift Card Redeem Start View"
+    __name__ = 'gift_card.redeem.start'
+
+    description = fields.Text('Description', required=True)
+    gateway = fields.Many2One(
+        'payment_gateway.gateway', 'Gateway', required=True,
+        domain=[
+            ('method', '=', 'gift_card'),
+        ]
+    )
+    gift_card = fields.Many2One(
+        'gift_card.gift_card', 'Gift Card', readonly=True
+    )
+    party = fields.Many2One('party.party', 'Party', required=True)
+    amount = fields.Numeric('Amount', digits=(16, 2), required=True)
+
+
+class GiftCardRedeemDone(ModelView):
+    "Gift Card Redeem Done View"
+    __name__ = 'gift_card.redeem.end'
+
+    done_msg = fields.Text("Message", readonly=True)
```

Comment:
No need to add a field. Just add a label in view.
Refer this https://github.com/tryton/trytond/blob/develop/trytond/ir/view/module_install_upgrade_done_form.xml#L6